molgenis-data-annotators/src/test/java/org/molgenis/data/annotation/entity/impl/SnpEffRunnerTest.java

```diff
@@ -302,4 +754,29 @@ public void getInputVcfFileTest()
 		}
 	}
 
+	@Test
+	public void getSnpEffectsTest()
+	{
+		try
+		{
+			List<String> params = Arrays.asList("-Xmx2g", null, "hg19", "-noStats", "-noLog", "-lof", "-canon", "-ud",
+					"0", "-spliceSiteSize", "5");
+			when(jarRunner.runJar(SnpEffAnnotator.NAME, params, new File("src/test/resources/test-snpeff.vcf")))
+					.thenReturn(new File("src/test/resources/snpEffOutput.vcf"));
+		}
+		catch (Exception e)
+		{
+			e.printStackTrace();
+		}
+
+		List<Entity> results = newArrayList(
+				snpEffRunner.getSnpEffects(entities.iterator(), new File("src/test/resources/test-snpeff.vcf")));
+
+		for (Entity resultEntity : results)
+		{
+			resultEntity.set("id", null);
+		}
+
+		assertEquals(results, expectedEffectEntities);
+	}
```

Comment:
Is it possible to split this test into multiple methods? One for single allele, one for multiple alleles, one for multiple genes and one for multiple genes and alleles? Then its a bit clearer what's being tested I think.